images/plbase/python-requirements.txt

```diff
@@ -1,13 +1,7 @@
 ansi2html==1.9.1
-black==24.3.0
 chevron==0.14.0
 coloraide==3.1.2
 Faker==24.4.0
-flake8-black==0.3.6
-flake8-isort==6.1.1
-Flake8-pyproject==1.2.3
-flake8==7.0.0
-isort==5.13.2
```

Comment:
Should we allow these to remain in case an instructor is relying on them being there in server.py? My instinct is no (these are more likely to be used in an autograder, not in question generation or internal grading), but I just want to make sure this is considered.

Reply:
I thought about this a little bit too, and I think it's super unlikely. I'm not even sure if flake8 or black are importable from a python script without doing something weird, they're really just command line applications.

Reply:
We definitely should not try to account for instructors trying to import these packages. Long-term I'd very much like to separate our dev + deployment images so that instructors can't use our dev packages even if they try.